src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/glide/MauiCustomViewTarget.java

```diff
@@ -46,11 +46,13 @@ public void onResourceReady(@NonNull Drawable resource, @Nullable Transition<? s
             return;
         this.completed = true;
 
-        // set the image
-        this.view.setImageDrawable(resource);
+        post(() -> {
+            // set the image
+            this.view.setImageDrawable(resource);
```

Comment:
This change to `post()` might be OK, did this fix something?

Reply:
Removing the ConfigureAwait(false) from StreamImageSourceService.Android.cs seemed to have worked by itself. The post() changes didn't fix anything on their own (i.e. I tested adding back the ConfigureAwait(false) and ended up with errors again).

Reply:
Maybe we don't need this anymore then? Is there any fears that there are related issues?